addons/web_editor/static/lib/odoo-editor/src/utils/constants.js

```diff
@@ -0,0 +1,3 @@
+/** @odoo-module **/
```

Comment:
Why single out these two? Why not also the pasting [black|white]lists, all those utils constants etc.?

Reply:
There were a cyclic dependency that confused the Odoo bundling code.